.github/workflows/ansible-playbooks/aio_tests.yml

```diff
@@ -0,0 +1,14 @@
+
+  - hosts: all
+    become: true
+
+    vars:
+      script_path: "{{ tmp_path }}/tests/install"
+      script_name: "{{ test_name }}.py"
+    
+    tasks:
+      - name: Test unattended AIO install
```

Comment:
Change the `unattended` references for `Installation assistant`.

Reply:
Done